README.md

```diff
@@ -50,99 +54,102 @@ libraryDependencies += "com.google.cloud" % "google-cloud-language" % "1.99.1"
 ```
 [//]: # ({x-version-update-end})
 
-Authentication
---------------
+## Authentication
 
-See the [Authentication](https://github.com/googleapis/google-cloud-java#authentication) section in the base directory's README.
+See the [Authentication][authentication] section in the base directory's README.
 
-About Google Cloud Natural Language
-----------------------------
+## Getting Started
 
-Google [Cloud Natural Language API][cloud-language] provides natural language understanding technologies to developers, including sentiment analysis, entity analysis, and syntax analysis. This API is part of the larger Cloud Machine Learning API family.
+### Prerequisites
 
-See the [Natural Language client library docs][language-client-lib-docs] to learn how to use this Cloud Natural Language API Client Library.
+You will need a [Google Cloud Platform Console][developer-console] project with the Natural Language [API enabled][enable-api].
+You will need to [enable billing][enable-billing] to use Google Natural Language.
+[Follow these instructions][create-project] to get your project set up. You will also need to set up the local development environment by
+[installing the Google Cloud SDK][cloud-sdk] and running the following commands in command line:
+`gcloud auth login` and `gcloud config set project [YOUR PROJECT ID]`.
 
-Getting Started
----------------
-#### Prerequisites
-You will need a [Google Developers Console](https://console.developers.google.com/) project with the Natural Language API enabled. [Follow these instructions](https://cloud.google.com/resource-manager/docs/creating-managing-projects) to get your project set up. You will also need to set up the local development environment by [installing the Google Cloud SDK](https://cloud.google.com/sdk/) and running the following commands in command line: `gcloud auth login` and `gcloud config set project [YOUR PROJECT ID]`.
+### Installation and setup
 
-#### Installation and setup
-You'll need to obtain the `google-cloud-language` library.  See the [Quickstart](#quickstart) section to add `google-cloud-language` as a dependency in your code.
+You'll need to obtain the `google-cloud-language` library.  See the [Quickstart](#quickstart) section
+to add `google-cloud-language` as a dependency in your code.
 
-#### Analyzing sentiment
-With Cloud Natural Language, you can analyze the sentiment of text. Add the following imports at the top of your file:
+## About Natural Language
 
-``` java
-import com.google.cloud.language.v1.LanguageServiceClient;
```

Comment:
did we mean to delete this?

Reply:
Restored by adding to the `.readme-partials.yaml` file